inc/api.php

```
@@ -337,7 +337,57 @@ function djz_get_gamipress_user_data($request) {
+                'limit' => -1,
+                'status' => ['completed', 'processing'],
+                'return' => 'ids',
+            ]);
+
+            if (!empty($orders)) {
+                foreach ($orders as $order_id) {
+                    $order = wc_get_order($order_id);
+                    if (!$order) continue;
+
+                    foreach ($order->get_items() as $item) {
+                        $product_id = $item->get_product_id();
```

PHPMD (2.15.0)

[error] The variable $product_id is not named in camelCase. (undefined)

(CamelCaseVariableName)